src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -18,19 +18,28 @@ class HoldRequest extends React.Component {
     super(props);
 
     const deliveryLocationsFromAPI = this.props.deliveryLocations;
+    const isEddRequestable = this.props.isEddRequestable;
     const firstLocationValue = (
       deliveryLocationsFromAPI.length &&
       deliveryLocationsFromAPI[0]['@id'] &&
       typeof deliveryLocationsFromAPI[0]['@id'] === 'string') ?
       deliveryLocationsFromAPI[0]['@id'].replace('loc:', '') : '';
+    let defaultDelivery = 'edd';
+    let checkedLocNum = -1;
+
+    // If EDD is available from the API, set EDD as the default delivery location,
```

Comment:
Since `defaultDelivery` and `checkedLocNum` are set to `edd` and `-1` by default, the wording here should not say `If ...`.